lighthouse-core/test/runner-test.js

```diff
@@ -625,8 +625,8 @@ describe('Runner', () => {
       assert.fail('should have thrown');
     } catch (err) {
       assert.equal(err.code, LHError.errors.PROTOCOL_TIMEOUT.code);
-      assert.ok(/^Waiting for DevTools protocol.*Method: Method.Failure/.test(err.friendlyMessage),
-        'should have prevented run');
+      /* eslint-disable-next-line max-len */
+      expect(err.friendlyMessage).toBeDisplayString('Waiting for DevTools protocol response has exceeded the allotted time. Method: Method.Failure.');
```

Comment:
wait no we really needed this to be the test that makes sure we're localizing without calling `toBeDisplayString` :D

Reply:
LOL I was wondering why it worked before!  This makes sense now.